docs/Migration-Guide-V3.md

```diff
@@ -32,6 +32,10 @@ await fastify.register(require('fastify-express'));
 fastify.use(require('cors')());
 ```
 
+**v4:**
+
+Starting v4 we have deprecated the use of `.use`. Both `middie` and `fastify-express` will still be there and maintained.
```

Comment:
Please start a Migration-Guide-V4.md file.

Reply:
Yeah this is the better way to do this. I am on it!